docs/rancher/cloud-provider.md

```diff
@@ -39,7 +39,6 @@ For a detailed support matrix, please refer to the **Harvester CCM & CSI Driver
 ### Prerequisites
 - The Kubernetes cluster is built on top of Harvester virtual machines.
 - The Harvester virtual machines run as guest Kubernetes nodes are in the same namespace.
-- The Harvester virtual machine guests' hostnames match their corresponding Harvester virtual machine names. Guest cluster Harvester VMs can't have different hostnames than their Harvester VM names when using the Harvester CSI driver. We hope [to remove this limitation](https://github.com/harvester/harvester/issues/4396) in a future release of Harvester.
```

Comment:
@brandboat can you also help figure out since which version we don't have this restriction anymore? Thanks!

Reply:
the fix was introduced in v1.3.0 

corresponding commit is here https://github.com/harvester/harvester-csi-driver/commit/d8da888088a977a3981fb5acc4784cdcffffa1b3 (in node-manager v0.1.6+)
in harvester v1.3.0 chart, node-manager version is v0.2.3 (https://github.com/harvester/harvester/blob/68c52fa87b44906b7915e81fb674fb73c1d41951/deploy/charts/harvester/Chart.yaml#L56)
in harvester v1.2.2 chart, node-manager version is v0.1.5 (https://github.com/harvester/harvester/blob/ccd7e1cf672b0608d1d50712b31c335d401105a0/deploy/charts/harvester/Chart.yaml#L56)

Reply:
I believe the csi driver version is relevant here

Reply:
Sorry... I'm not sure what I was thinking then, will edit comment later. Thanks

Reply:
the fix was introduced since harvester-csi-driver release v0.1.6 https://github.com/harvester/harvester-csi-driver/commit/d8da888088a977a3981fb5acc4784cdcffffa1b3

And was included in rke2 versions:

rke2 1.31 `>=` [v1.31.0+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.31.0%2Brke2r1), commit:  https://github.com/rancher/rke2/commit/7a81df0886ca523981ba3aceb4301124a08eb24f

rke2 1.30 `>=` [v1.30.0+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.30.0%2Brke2r1), commit:  https://github.com/rancher/rke2/commit/7a81df0886ca523981ba3aceb4301124a08eb24f

rke2 1.29 `>=` [v1.29.2+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.29.2%2Brke2r1), commit:  https://github.com/rancher/rke2/commit/7a81df0886ca523981ba3aceb4301124a08eb24f

rke2 1.28 `>=` [v1.28.7+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.28.7%2Brke2r1) , commit: https://github.com/rancher/rke2/commit/42eefe402823d3a5d89a31b9fa934db1a164bff5 

rke2 1.27 `>=` [v1.27.11+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.27.11%2Brke2r1) , commit: https://github.com/rancher/rke2/commit/335307711eaac128e10eafab8ea375c135ebfdb3

rke2 1.26 `>=` [v1.26.14+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.26.14%2Brke2r1) , commit: https://github.com/rancher/rke2/commit/0b6e0d8737b894f6d77ff7a124082d7c1f3a9659